packages/certs-v5/commands/certs/add.js

```diff
@@ -230,11 +230,13 @@ function * configureDomains (context, heroku, meta, cert) {
 }
 
 function * run (context, heroku) {
+  let features = yield heroku.get(`/apps/${context.app}/features`)
+  let canMultiSni = !!features.find(feature => feature.name === 'allow-multiple-sni-endpoints' && feature.enabled === true)
```

Comment:
Very minor but you could use `.some()` instead of `.find()` and remove the need for the `!!`:
```suggestion
  let canMultiSni = features.some(feature => feature.name === 'allow-multiple-sni-endpoints' && feature.enabled === true)
```

Reply:
good point!